### ISSUE TITLE
[Bug]: Agent timeout for no reason

### ISSUE THREAD

**Issue by cakrit** [netdata/netdata-cloud#488]
### Bug description

Multiple data requests to the following return data only from one of the two nodes in the room. However, each node on its own responds well and netdata is running fine on both of them. 
 
https://app.netdata.cloud/api/v2/spaces/47e025ed-c1d5-4981-b170-3cb4d4a56a9a/rooms/0852b034-7062-43a0-8ea5-58b9389dad51/data
```
{"filter":{"nodeIDs":["1065d4da-410a-4921-8630-b7c414756ba1","3b3d712b-e3f6-478a-976e-c339ee206e83"],"context":"system.cpu_full_pressure"},"aggregations":[{"method":"avg","groupBy":["dimension"]}],"agent_options":["jsonwrap","flip","ms"],"points":535,"format":"json","group":"","gtime":0,"after":-300,"with_metadata":true}
```

Response contains the following:
```    "nodes": [
        {
            "id": "1065d4da-410a-4921-8630-b7c414756ba1",
            "routing_latency": "2.032299ms",
            "adc_latency": "87.435018ms",
            "chartIDs": [
                "system.cpu_full_pressure"
            ],
            "hops": 0,
            "coverage": 0
        },
        {
            "id": "3b3d712b-e3f6-478a-976e-c339ee206e83",
            "routing_latency": "1.881936ms",
            "adc_latency": "10.003415746s",
            "error": {
                "errorMsgKey": "ErrAgentTimeout",
                "errorMessage": "timeout waiting agent response",
                "errorCode": "89pv663WdG-15794831"
            },
            "chartIDs": [],
            "hops": 1,
            "coverage": 0
        }
    ],
```

Verified that the nodes view also gets data for the same node id, `3b3d712b-e3f6-478a-976e-c339ee206e83`

### Expected behavior

A node either responds or doesn't, this seems to be a BE bug

### Steps to reproduce

Dunno, I can provide access to the room, ask me in Slack.

### Desktop

Browser Version [e.g. 22]

**Comment by hugovalente-pm** [comment]
what I can see on the Overview is that one node is being marked has having timedout

![image](https://user-images.githubusercontent.com/82235632/176450268-8989a6ef-4d00-47b2-a597-aa3a4fa148fa.png)

as mentioned, if you go to single node view you get data for that node

for the one that timesout you see on the Overview request:
```
			"chartIDs": [],
			"hops": 1,
			"coverage": 0
```

but on the single node view you see data being retrieved from the node itself
```
	"meta": {
		"coverage": 0,
		"hops": 0
	},
```

**Comment by papazach** [comment]
Hello, I released the fix addressing this bug to production.

A specific part of the context data queries routing logic had been temporarily deactivated until we enhanced it with a light caching layer to ease the load to the DB. So this has been applied and from the telemetry there seems to be no actual performance penalty.

Feel free to validate as well and close the ticket.

@cakrit @hugovalente-pm
